### PR TITLE
Feature/DAG2

### PR DESCRIPTION
**Pull request by mblomdahl**
Work in progress.. Switching to Conda, adding own Hooks and Sensors for HDFS.

### CODE REVIEW

**Comment by carlba** [comment]
Would be nice to know the reason for switching to anaconda. What was hard to accomplish without out.

**Comment by mblomdahl** [comment]
screenshot 2019-01-27 at 16 28 46

Now it works! Removing WIP status. :)

File: o3/hooks/hdfs_hook.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Copy of :py:mod:`airflow.hooks.hdfs_hook`, modified to replace Py3 incompatible
+snakebite library wrapping with libhdfs3 driver.
+"""
+
+from airflow.hooks.base_hook import BaseHook
+
+import hdfs3
```
Comment (mblomdahl) [inline o3/hooks/hdfs_hook.py]: @carlba This was the thing that was so damn hard getting working without the Conda packages. If you find a neater solution, I'd be happy to know about it! I stopped using Anaconda +3 years ago in favor of standard PyPI and stuff. A shame having to go back...

Comment (mblomdahl) [inline o3/hooks/hdfs_hook.py]: @carlba This was the thing that was so damn hard getting working without the Conda packages. If you find a neater solution, I'd be happy to know about it! I stopped using Anaconda +3 years ago in favor of standard PyPI and stuff. A shame having to go back...

**Pull request merged**